ui/overviewTab/overviewTab.pug

```diff
@@ -0,0 +1,107 @@
+// Overview section
+.overview.section
+  .horizontal-flex
+    // Advantages
+    .flex-stack.advantage-overview.no-add-or-modify-buttons
+      .table-header
+        div Advantage / Disadvantage
+      .conditional-something
+        input(name='attr_advantage_healthy_count' type='hidden')
+        div
+          | Healthy
+          | #[span(name='attr_advantage_healthy_count')]
+      .conditional-something
+        input(name='attr_advantage_packmule_count' type='hidden')
+        div
+          | Pack Mule
+          | #[span(name='attr_advantage_packmule_count')]
+      .conditional-something
+        input(name='attr_advantage_strongarms_count' type='hidden')
+        div
+          | Strong Arms
+          | #[span(name='attr_advantage_strongarms_count')]
+      .conditional-something
+        input(name='attr_advantage_brute_count' type='hidden')
+        div
+          | Brute
+          | #[span(name='attr_advantage_brute_count')]
+      .conditional-something
+        input(name='attr_advantage_fast_count' type='hidden')
+        div
+          | Fast
+          | #[span(name='attr_advantage_fast_count')]
+      .conditional-something
+        input(name='attr_advantage_athlete_count' type='hidden')
+        div
+          | Natural Athlete
+          | #[span(name='attr_advantage_athlete_count')]
+      .conditional-something
+        input(name='attr_advantage_alert_count' type='hidden')
+        div
+          | Alert
+          | #[span(name='attr_advantage_alert_count')]
+      .conditional-something
+        input(name='attr_advantage_wary_count' type='hidden')
+        div
+          | Wary
+          | #[span(name='attr_advantage_wary_count')]
+      .conditional-something
+        input(name='attr_advantage_mage_count' type='hidden')
+        div
+          | Mage
+          | #[span(name='attr_advantage_mage_count')]
+      .conditional-something
+        input(name='attr_advantage_devout_count' type='hidden')
+        div
+          | Devout
+          | #[span(name='attr_advantage_devout_count')]
```

Comment:
@johnlamping here is an example of what pug can do to replace line 8 - 57:

```
-
  var advantages = {
    'attr_advantage_healthy_count': 'Healthy',
    'attr_advantage_packmule_count': 'Pack Mule',
    'attr_advantage_strongarms_count': 'Strong Arms',
    'attr_advantage_brute_count': 'Brute',
    'attr_advantage_fast_count': 'Fast',
    'attr_advantage_athlete_count': 'Natural Athlete',
    'attr_advantage_alert_count': 'Alert',
    'attr_advantage_wary_count': 'Wary',
    'attr_advantage_mage_count': 'Mage',
    'attr_advantage_devout_count': 'Devout',
  }
```
and
```
      each advantageDescription, advantageName in advantages
        .conditional-something
          input(name=advantageName type='hidden')
          div
            | = advantageDescription
            | #[span(name=advantageName)]
```

I'll push a delta PR that shows this.

Reply:
Merged into this PR. It works nicely now.